angular_components/lib/simple_html/README.md

```diff
@@ -0,0 +1,297 @@
+
+# Simple HTML
+
+Displays low-trust HTML with strict and flexible sanitization.
+
+This component is focused on displaying localized or otherwise partially-trusted
+HTML in the following use cases:
+
+*   Light formatting (bold, italic, newlines).
+*   Links to external domains (with a customisable whitelist).
+*   Arbitrary click handlers on substrings within the text.
+
+Mixing these features with text that cannot appear verbatim in the template (due
+to, for example, use of the `Intl` library for localization) is otherwise
+difficult. This component is the recommended alternative to calling methods like
+[`bypassSecurityTrustHtml`](https://github.com/dart-lang/angular/blob/master/angular/lib/src/security/dom_sanitization_service.dart#L53)
```

Comment:
- `bypassSecurityTrustHtml` -> `bypassSecurityTrustHtml()`
- Linking to the code is brittle. Consider linking to the corresponding API doc entry: https://pub.dartlang.org/documentation/angular/latest/angular.security/DomSanitizationService/bypassSecurityTrustHtml.html